src/components/charts/shared/navDrawer/nav-drawer.module.scss

```diff
@@ -1,6 +1,7 @@
 .nav-drawer-container {
   position: relative;
   overflow: hidden;
+  z-index: 1;
```

Comment:
z-index should come before overflow (see order/properties-order linter warning in terminal)

Reply:
ran `yarn styles`